stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetListFragment.kt

```diff
@@ -25,24 +22,9 @@ internal class PaymentSheetListFragment(
 
     override val sheetViewModel: PaymentSheetViewModel by lazy { activityViewModel }
 
-    private val currencyFormatter = CurrencyFormatter()
-
     override fun transitionToAddPaymentMethod() {
         activityViewModel.transitionTo(
             PaymentSheetViewModel.TransitionTarget.AddPaymentMethodFull(config)
         )
     }
-
-    override fun createHeaderText(): String {
-        val paymentIntent = config.paymentIntent
-        return if (paymentIntent.currency != null && paymentIntent.amount != null) {
-            val currency = Currency.getInstance(paymentIntent.currency.toUpperCase(Locale.ROOT))
-            getString(
-                R.string.stripe_paymentsheet_pay_using_with_amount,
-                currencyFormatter.format(paymentIntent.amount, currency)
-            )
-        } else {
-            getString(R.string.stripe_paymentsheet_pay_using)
-        }
-    }
```

Comment:
Why are we removing this?

Reply:
To match the iOS version. The amount is not shown on the header, only on the button.